pacman_mirrors/mirror.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+#
+# This file is part of pacman-mirrors.
+#
+# pacman-mirrors is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# pacman-mirrors is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with pacman-mirrors.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Authors   : Frede Hundewadt <frede@hundewadt.dk>
+
+"""Pacman-Mirrors Mirror"""
+
+from operator import itemgetter
+from random import shuffle
+
+
+class Mirror:
+    """Mirror Class"""
+
+    def __init__(self):
+        self.countrylist = []
+        self.mirrorlist = []
+
+    def add(self, country, url, protocols, branches=None, last_sync="00:00", resp_time="00.00"):
+        """Append mirror
+        :param country:
+        :param url:
+        :param protocols:
+        :param branches: optional from status.json
+        :param last_sync: optional from status.json
+        :param resp_time: optional from status.json
+        """
+        if branches is None:
+            branches = [0, 0, 0]
+        if country not in self.countrylist:
+            self.countrylist.append(country)
+        # workaround for negative integer in status.json
```

Comment:
It's not a workaround. For me, it's fine: just the translation of a parameter to get what we want.